app/views/teacher_training_adviser/steps/_primary_returner.erb

```diff
@@ -5,7 +5,7 @@
     <div class="govuk-grid-column-two-thirds">
       <h1 class="govuk-heading-l">We're sorry, but you are not eligible for this service.</h1>
 
-      <p>To be eligible, you must be planning to teach chemistry, computing, maths, modern foreign languages or physics.</p>
+      <p>To be eligible, you must be planning to teach a secondary subject.</p>
```

Comment:
```suggestion
      <p>If you already have qualified teacher status (QTS) and are returning to teaching, you must be planning to teach a secondary subject to be eligible.</p>
```
@gemmadallmandfe do we think it might be good to be clearer here? Just in case someone has accidentally navigated to this end screen, then thinks **all** candidates need to be teaching a secondary subject to get an adviser, not just returning teachers.

Reply:
@EllieNodder Yes, I think that would make sense

Maybe '.... to be eligible for an adviser.'? The page is a bit odd as it stands as it talks about being eligible for the service but doesn't really say what it is - is that deliberate?